src/redux/reducers/experimentSettings/updateExperimentInfo.js

```diff
@@ -17,7 +17,7 @@ const updateExperimentInfo = produce((draft, action) => {
   draft.info.sampleIds = sampleIds;
 
   // Experiment id was updated so processing config requires reloading
```

Comment:
is this comment still relevant?

Reply:
I think it is, we are wiping the processing config instead of setting `loading: true`, but wiping it causes the ui to allow `loadProcessingConfig` (just like before setting loading to true did)

Reply:
there's a small explanation in "Anything else the reviewers should know about the changes here"

Reply:
indeed